app/assets/javascripts/shared/dradis_datatable.js

```diff
@@ -73,6 +82,126 @@ class DradisDatatable {
     this.unbindDataTable();
   }
 
+  bulkDelete() {
+    var that = this;
+    var destroyConfirmation = that.$paths.data('table-destroy-confirmation') || 'Are you sure?';
+    var answer = confirm(destroyConfirmation);
+
+    if (!answer) {
+      return;
+    }
+
+    var destroyUrl = that.$paths.data('table-destroy-url');
+    var selectedRows = that.dataTable.rows({ selected: true });
+    that.toggleBulkDeleteLoadingState(selectedRows, false);
```

Comment:
Is this the best place to re-enable the button + remove the loading state? Shouldn't this be done on the ajax's `.always()` callback?